app/bingey-ui/src/components/Watchlist/TitleCard/TitleCard.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+
+import 'components/Watchlist/TitleCard/cardStyle.scss';
+
+export default function TitleCard({ titles }) {
+  const [open, setOpen] = useState(false);
+
+  const handleOpenDialog = () => {
+    setOpen(true);
+  };
+
+  const handleCloseDialog = () => {
+    setOpen(false);
+  };
+
+  return (
+    <div>
+      <Card>
```

Comment:
I'd suggest
```suggestion
      <Card elevation={5}>
```
Currently the shadow looks terribly ugly imo